Bugzilla/Elastic/Indexer.pm

```diff
@@ -109,7 +104,7 @@ sub _bulk_helper {
     );
 }
 
-sub find_largest_mtime {
```

Comment:
It was because a previous review asked for sanity checking a value, and a desire to not repeat myself. So I merged two code paths into one implementation